base/div.jl

```diff
@@ -67,6 +67,7 @@ rem(x, y, r::RoundingMode)
 rem(x, y, ::RoundingMode{:ToZero}) = rem(x,y)
 rem(x, y, ::RoundingMode{:Down}) = mod(x,y)
 rem(x, y, ::RoundingMode{:Up}) = mod(x,-y)
+rem(x, y, r::RoundingMode{:Nearest}) = x - y*div(x, y, r)
```

Comment:
Since this `div` is defined in terms of `divrem`, it would make sense to use it for `rem` as well.
```suggestion
rem(x, y, r::RoundingMode{:Nearest}) = divrem(x, y, r)[2]
```